cernopendata/modules/fixtures/data/records/delphi-author-list-all.json

```diff
@@ -0,0 +1,4427 @@
+        },
+        {
+          "name": "Niezurawski, P.",
+          "affiliation": "Inst. Nuclear Studies and University of Warsaw, Ul. Hoza 69, PL-00681 Warsaw, Poland"
+        },
+        {
+          "name": "Nijjhar, B.",
+          "affiliation": "Department of Physics, University of Liverpool, P.O. Box 147, Liverpool L69 3BX, UK"
+        },
+        {
+          "name": "Nikolaenko, V.",
+          "affiliation": "CERN, CH-1211 Geneva 23, Switzerland; Institut de Recherches Subatomiques, IN2P3 - CNRS-ULP - BP20, FR-67037 Strasbourg Cedex, France; Universite Claude Bernard de Lyon, IPNL, IN2P3-CNRS, FR-69622 Villeurbanne Cedex, France; Inst. for High Energy Physics, Serpukov P.O. Box 35, Protvino, (Moscow Region), Russian Federation"
+        },
+        {
+          "name": "Nikolenko, M.",
+          "affiliation": "Institut de Recherches Subatomiques, IN2P3 - CNRS-ULP - BP20, FR-67037 Strasbourg Cedex, France; Joint Institute for Nuclear Research, Dubna, Head Post Office, P.O. Box 79, RU-101 000 Moscow, Russian Federation"
+        },
+        {
+          "name": "Nilsen, P.E.S.",
+          "affiliation": "Department of Physics, University of Bergen, Allegaten 55, NO-5007 Bergen, Norway"
+        },
+        {
+          "name": "Niss, P.",
+          "affiliation": "Fysikum, Stockholm University, Box 6730, SE-113 85 Stockholm, Sweden"
+        },
+        {
+          "name": "Nomerotski, A.",
+          "affiliation": "Dipartimento di Fisica, Universita di Padova; INFN, Via Marzolo 8, IT-35131 Padua, Italy;FERMILAB"
+        },
+        {
+          "name": "Nomokonov, V.",
+          "affiliation": "Helsinki Institute of Physics, HIP, P.O. Box 9, FI-00014 Helsinki, Finland"
+        },
+        {
+          "name": "Norman, A.",
+          "affiliation": "Department of Physics, University of Oxford, Keble Road, Oxford OX1 3RH, UK"
+        },
+        {
+          "name": "Normand, A.",
+          "affiliation": "Department of Physics, University of Liverpool, P.O. Box 147, Liverpool L69 3BX, UK; Department of Physics, University of Oxford, Keble Road, Oxford OX1 3RH, UK"
+        },
+        {
+          "name": "Novak, M.",
+          "affiliation": "FZU, Inst. of Phys. of the C.A.S. High Energy Physics Division, Na Slovance 2, CZ-180 40, Praha 8, Czech Republic"
+        },
+        {
+          "name": "Novak, N.",
+          "affiliation": "FZU, Inst. of Phys. of the C.A.S. High Energy Physics Division, Na Slovance 2, CZ-180 40, Praha 8, Czech Republic"
+        },
+        {
+          "name": "Nygren, A.",
+          "affiliation": "Department of Physics, University of Lund, Solvegatan 14, SE-223 63 Lund, Sweden"
+        },
+        {
+          "name": "Oberschulte-Beckmann, W.",
+          "affiliation": "Institut fuer Experimentelle Kernphysik, Universitaet Karlsruhe, Postfach 6980, DE-76128 Karlsruhe, Germany"
+        },
+        {
+          "name": "Oberschulte~gen.~Beckmann, W.",
```

Comment:
`Oberschulte~gen.~Beckmann, W.` and `Oberschulte-Beckmann, W.` are probably the same person?